app/javascript/react/components/NewReviewContainer.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+
+import NewReviewForm from './NewReviewForm'
+
+const NewReviewContainer = props => {
+  const game_id = props.game_id
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const handleFormSubmit = formData => {
+    formData.game_id = game_id
+    fetch('/api/v1/reviews', {
```

Comment:
this is not the correct path. reviews should be nested under games so that the game_id is automatically in your params via the url. 

you shouldnt need to manually add the game id here either.